skies_of_land/map.conf

```
@@ -0,0 +1,21 @@
+game_modes = return {"classic"}
+barrier_area = return {pos1={y=79,x=111,z=221},pos2={y=1,x=111,z=1}}
+teams = local _={};_[1]="flag_pos";_[2]="enabled";return {orange={pos2={y=79,x=1,z=221},[_[2]]=true,pos1={y=1,x=111,z=1},[_[1]]={y=46,x=21,z=111}},blue={pos2={y=1,x=111,z=1},[_[2]]=true,pos1={y=79,x=221,z=221},[_[1]]={y=46,x=201,z=111}}}
+chests = return {{pos2={y=40,x=155,z=79},pos1={y=38,x=143,z=67},amount=18},{pos2={y=40,x=155,z=155},pos1={y=38,x=143,z=143},amount=18},{pos2={y=40,x=79,z=155},pos1={y=38,x=67,z=143},amount=18},{pos2={y=40,x=79,z=79},pos1={y=38,x=67,z=67},amount=18},{pos2={y=49,x=209,z=209},pos1={y=47,x=201,z=201},amount=30},{pos2={y=49,x=21,z=21},pos1={y=47,x=13,z=13},amount=30}}
+phys_jump = 1
+phys_speed = 1.2
+size = return {y=80,x=222,z=222}
+map_version = 2
+enabled = true
+name = Skies of Land
+phys_gravity = 1
+initial_stuff = return {"default:cobble 10","default:torch 5","default:stick 5","ctf_ranged:pistol_loaded 1"}
+author = daveistaken
+hint = Search the sandwell outside your base for ores buried in the sand. For even more valuable ores, check beneath in the heart of the lighthouse.
+license = CC BY-SA 4.0 | Hint is reworked with https://poe.com and it's edited by me.
```

```suggestion
license = CC BY-SA 4.0
```
I don't think it's needed. (*citation needed)

---

If you want to add it to the `Other Info`, then I guess it's your wish :thumbsup:

---

I copied the hint to the website and then told AI to rework it (making it correct), also if that didn't exist in the license, it would be confusing if it was either written by human or 'artificial intelligence', surely it's necessary, but I won't remove it (just to give extra credit)

---

> @src4026 I copied the hint to the website and then told AI to rework it (making it correct), also if that didn't exist in the license, it would be confusing if it was either written by human or 'artificial intelligence', surely it's necessary, but I won't remove it (just to give extra credit)

> If generative AI output is considered a derivative work of the training materials, engineering teams that use it would be required to comply with the license(s) of the code upon which the tool is trained. This, of course, could come with requirements to disclose source code, generate attribution notices, and more.
> But this is only the case if generative AI output contains copyrightable expression. And, although the book isn’t closed on this question, there are considerations that suggest it doesn’t. Sufficiently short phrases — which generative AI often produces in the software development context — are not copyrightable and therefore neither subject to copyright nor any open source license.

From https://fossa.com/blog/generative-ai-and-software-development-copyright-law-and-license-compliance/